tests/radius-v.out

```diff
@@ -9,7 +9,8 @@
 	  Calling-Station-Id Attribute (31), length: 19, Value: 00-14-22-E9-54-5E
 	  Service-Type Attribute (6), length: 6, Value: Framed
 	  Framed-MTU Attribute (12), length: 6, Value: 1500
-	  EAP-Message Attribute (79), length: 19, Value: .
+	  EAP-Message Attribute (79), length: 19, Value: , Response (2), id 0, len 17
```

Comment:
Why `Value: ` (empty)

Reply:
Actually, the EAP Attribute is in itself a TLV and hence, the value cannot be printed as such like a string/number. 
Hence, previously . was printed as a result. 

Now, with the new changes, the TLV is extracted and contents are printed. I previously wanted to separate out the extracted part and the value part and hence, value was empty.

But, as you pointed out, it makes more sense to avoid printing empty value and print the extracted contents directly.
Addressed the same.

Thanks a lot for the extensive review.